src/bootstrap/mk/Makefile.in

```diff
@@ -45,6 +45,10 @@ check-aux:
 		src/tools/cargo \
 		src/tools/cargotest \
 		$(BOOTSTRAP_ARGS)
+check-aux-and-gui: check-aux
```

Comment:
It looks like you didn't actually make use of this make rule anywhere? I believe you'll want to set RUST_CHECK_TARGET in the docker file (i.e., edit the existing declaration) to check-aux-and-gui.

Reply:
Added.